duckduckgo/macos-browser
# remove red dot for major trackers

Task/Issue URL: https://app.asana.com/0/1177771139624306/1204128430184670/f
Tech Design URL:
CC:

**Description**:

**Steps to test this PR**:
1. Visit Facebook.com
2. Confirm no red dot

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
